pkg/client/didexchange/client_test.go

```diff
@@ -136,20 +137,20 @@ func TestClient_CreateInvitationWithDID(t *testing.T) {
 }
 
 func TestClient_QueryConnectionByID(t *testing.T) {
+	connID := "id1"
```

Comment:
if these are constants you can declare them as constants (for this test)

Reply:
done